content/en/blog/_posts/Deploying-External-OpenStack-Cloud-Provider-With-Kubeadm.md

```diff
@@ -0,0 +1,760 @@
+---
+layout: blog
+title: "Deploying External OpenStack Cloud Provider with Kubeadm"
+date: 2020-01-20
+slug: Deploying-External-OpenStack-Cloud-Provider-with-Kubeadm
+---
+This document describes how to install a single-master Kubernetes cluster v1.15 with kubeadm on CentOS, and then deploy an external OpenStack cloud provider and Cinder CSI plugin to use Cinder volumes as persistent volumes in Kubernetes.
+
+### Preparation in OpenStack
+
+This cluster will be running on OpenStack VMs so we'll create a few things in OpenStack for it.
+
+* A project/tenant for this Kubernetes cluster
+* a user in this project for Kubernetes, to query node information and attach volumes etc
```

Comment:
```suggestion
* A user in this project for Kubernetes, to query node information and attach volumes.
```

Reply:
nit: can you capitalize the elements in these bullet points?

Reply:
fixed